apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -315,8 +315,23 @@ export default function Success(props: PageProps) {
       return t(`needs_to_be_confirmed_or_rejected${titleSuffix}`);
     }
     if (bookingInfo.user) {
-      return t(`${titlePrefix}emailed_you_and_attendees${titleSuffix}`, {
-        user: bookingInfo.user.name || bookingInfo.user.email,
+      const isHost = bookingInfo.user.id === session?.user?.id;
+      const isAttendee = bookingInfo.attendees.find((attendee) => attendee.email === session?.user?.email);
+      const attendee = bookingInfo.attendees[0].name || bookingInfo.attendees[0].email;
+      const host = bookingInfo.user.name || bookingInfo.user.email;
+      if (isHost) {
+        return t(`${titlePrefix}emailed_you_and_attendees${titleSuffix}`, {
+          user: attendee,
+        });
+      }
+      if (isAttendee) {
+        return t(`${titlePrefix}emailed_you_and_attendees${titleSuffix}`, {
+          user: host,
+        });
+      }
+      return t(`${titlePrefix}emailed_host_and_attendee${titleSuffix}`, {
+        host,
+        attendee,
```

Comment:
For guests: You are meeting with Host Name
For Hosts: You are meeting with Guest name
For Admins/Owners and for any user that are not part of the booking: Host Name is meeting with Guest Name